.github/workflows/deploy.yml

```diff
@@ -0,0 +1,89 @@
+#
+# Copyright 2018 ABSA Group Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: Deploy
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag-name:
```

Comment:
You probably don't need that. You can just select the tag from the dropdown
<img width="316" alt="image" src="https://github.com/user-attachments/assets/a2c60c98-67fd-44b3-80d3-9eb716c82a2a" />

Then you don't need to validate the input